src/js/media-controller.js

```diff
@@ -629,36 +836,33 @@ class MediaController extends MediaContainer {
   mediaSetCallback(media) {
     super.mediaSetCallback(media);
 
+    // TODO: What does this do? At least add comment, maybe move to media-container
```

Comment:
@gkatsev what does this do?

Reply:
This makes the media element focusable if a user didn't specify a tabindex already. -1 means that it'll be focusable via a mouse click or programmatically, but not be part of the tab order.

I believe it's necessary to keep focus inside the player for things like keyboard shortcuts.